packages/fxa-content-server/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 1.166.0
+
+### New features
+
+- payment-server: mobile payments flow for fpn ([33e59c64d](https://github.com/mozilla/fxa/commit/33e59c64d))
+- admin-panel: Add server_config, apollo-client, graphQL query and handling of response, favicon, style improvements, add script for new email bounce ([c4b05a435](https://github.com/mozilla/fxa/commit/c4b05a435))
```

Comment:
payment server?  admin panel?  This is ~Sparta~ content server!

Reply:
The commit did touch content-server so maybe that is why?

Reply:
Yeah, looks like the favicon in content server was accidentally removed.